contents/ruby/Ruby.md

```diff
@@ -0,0 +1,142 @@
+# Introduction to Ruby
+
+Authors: [Wilson Kurniawan](https://github.com/wkurniawan07)
+
+## Ruby Overview
+
+**Ruby** is a **dynamic**, **reflective**, **object-oriented**, **general-purpose** programming-language, designed and developed by Yukihiro "Matz" Matsumoto in 1990s.
+It is widely used across many fields of computer science and software engineering, and its most prevalent use is in web development.
+
+Ruby boasts high programmer productivity with its concise, elegant, and human-readable syntax, which is widely agreed to outweigh the loss in execution speed (as compared to compiled languages such as C and Java) unless the latter is critical.
+
+> Often people, especially computer engineers, focus on the machines. They think, "By doing this, the machine will run faster. By doing this, the machine will run more effectively. By doing this, the machine will something something something."
+> They are focusing on machines. But in fact, we need to focus on humans, on how humans care about doing programming or operating the application of the machines. We are the masters. They are the slaves.
+> 
+> *Yukihiro Matsumoto, creator of Ruby language* 
+
+Popular websites running on Ruby (or its framework): GitHub, Airbnb, Twitter, Shopify.
```

Comment:
Readers might not know what it means by `(or its framework)`. You can either cut out the three words, or you can briefly explain that Ruby on Rails is a popular framework written in Ruby.